test/binaries/test-ncurses-6.1.c

```diff
@@ -0,0 +1,12 @@
+#include <stdio.h>
+
+int main() {
+	printf("This program is designed to test the cve-bin-tool checker.");
+	printf("It outputs a few strings normally associated with ncurses-6.1");
+	printf("They appear below this line.");
+	printf("------------------");
+	printf("NCURSES");
+	printf("ncurses 6.1");
```

Comment:
Could you change this to be a version that doesn't match the real file you're testing below?  (So basically anything other than 6.1 here, or change the test below to be somethign other than 6.1)  We've got a bug that's causing the one test result to mess up the other in test_scanner and I want to be 100% sure that this is working as intended.